clients/fides-js/src/lib/overlay.module.css

```diff
@@ -2,61 +2,49 @@
-  --fides-consent-overlay-font-size-title: 18px;
-  --fides-consent-overlay-background: #fff;
-  --fides-consent-overlay-text-color: #24292f;
-  --fides-consent-overlay-padding: 0.75em 2em 1em;
-
-  --fides-consent-overlay-button-background: #eee;
-  --fides-consent-overlay-button-text-color: #24292f;
-  --fides-consent-overlay-button-hover-background: #ccc;
-  --fides-consent-overlay-button-border-radius: 4px;
-  --fides-consent-overlay-button-padding: 0.5em 1em;
-
-  --fides-consent-overlay-button-primary-background: #464b83;
-  --fides-consent-overlay-button-primary-text-color: #fff;
-  --fides-consent-overlay-button-primary-hover-background: #3f4375;
-
-  --fides-consent-overlay-button-secondary-background: var(
-    --fides-consent-overlay-button-background
-  );
-  --fides-consent-overlay-button-secondary-text-color: var(
-    --fides-consent-overlay-button-text-color
-  );
-  --fides-consent-overlay-button-secondary-hover-background: var(
-    --fides-consent-overlay-button-hover-background
-  );
-
-  --fides-consent-overlay-button-tertiary-background: var(
-    --fides-consent-overlay-button-background
-  );
-  --fides-consent-overlay-button-tertiary-text-color: var(
-    --fides-consent-overlay-button-text-color
-  );
-  --fides-consent-overlay-button-tertiary-hover-background: var(
-    --fides-consent-overlay-button-hover-background
-  );
-
-  /* In progress(fides#3258): start to transition to updated design colors */
+  /* Colors */
   --fides-consent-overlay-primary-color: #8243f2;
   --fides-consent-overlay-background-color: #f7fafc;
+  /* Buttons */
+  --fides-consent-overlay-primary-button-background-color: #824ef2;
+  --fides-consent-overlay-primary-button-background-hover-color: #9569f4;
+  --fides-consent-overlay-primary-button-text-color: #ffffff;
+  --fides-consent-overlay-primary-button-border-color: transparent;
+  --fides-consent-overlay-secondary-button-background-color: #f7fafc;
+  --fides-consent-overlay-secondary-button-background-hover-color: #edf2f7;
+  --fides-consent-overlay-secondary-button-text-color: #2d3748;
+  --fides-consent-overlay-secondary-button-border-color: #824ef2;
+  /* Text */
+  --fides-consent-overlay-title-font-color: #4a5568;
+  --fides-consent-overlay-body-font-color: #4a5568;
+  /* Switches */
   --fides-consent-overlay-primary-active-color: #824ef2;
   --fides-consent-overlay-primary-active-disabled-color: #bda4f7;
   --fides-consent-overlay-inactive-color: #cbd5e0;
   --fides-consent-overlay-disabled-color: #e1e7ee;
+  /* Dividers */
   --fides-consent-overlay-row-divider-color: #e2e8f0;
   --fides-consent-overlay-row-hover-color: #edf2f7;
+
+  /* Everything else */
+  --fides-consent-overlay-font-family: "inherit";
+  --fides-consent-overlay-font-size: 16px;
+  --fides-consent-overlay-font-size-title: 18px;
+  --fides-consent-overlay-padding: 0.75em 2em 1em;
+  --fides-consent-overlay-button-border-radius: 4px;
+  --fides-consent-overlay-button-padding: 0.5em 1em;
```

Comment:
these already existed but not all of them were explicitly called out in the design—I kept them around since they seemed useful